DwarfCorp/DwarfCorpXNA/GameStates/OptionsState.cs

```diff
@@ -548,7 +548,7 @@ private void CreateGameplayTab()
                 OnSliderChanged = (widget) =>
                 {
                     AutoSaveFrequency.GetChild(0).Text = String.Format("Autosave Frequency ({0})",
-                        (widget as HorizontalSlider).ScrollPosition + 5);
+                        (widget as HorizontalSlider).ScrollPosition + 1);
```

Comment:
This limits the minimum autosave frequency to 1 minute. Are you sure that's not too fast?